client/views/admin/rooms/EditRoom.js

```diff
@@ -142,6 +150,7 @@ function EditRoom({ room, onChange }) {
 
 	const changeRoomType = useMutableCallback(() => {
 		handleRoomType(roomType === 'p' ? 'c' : 'p');
+		handleRoomTags([]);
```

Comment:
It would better here to only disable the tags field and do a check when saving to clear the tags if the room type is `p`, this can lead to some frustration, also because is possible to change this field with the tags field out of view (if the user has scrolled to the end)